src/v2/view-builder/utils/AuthenticatorUtil.js

```diff
@@ -96,4 +97,18 @@ export function getAuthenticatorDataForVerification (authenticator) {
 
 export function getIconClassNameForBeacon (authenticatorType) {
   return getAuthenticatorData({ authenticatorType }).iconClassName;
-}
\ No newline at end of file
+}
+
+export function removeRequirementsFromError (errorJSON, policy) {
+  const passwordRequirementsAsString = getPasswordComplexityDescription(policy);
+  if (errorJSON.errorCauses?.length > 0
+    && Array.isArray(errorJSON.errorCauses[0].errorSummary)
```

Comment:
Can we do the same logic here?

```
errorJSON.errorCauses?.length > 0 && errorJSON.errorCauses[0].errorSummary?.length > 0
```

Reply:
`Array.isArray` would be a better check since length prop exists on strings too. I'm also indexing into the array on the next line.

Reply:
Ah that makes sense, would we want the same check for `errorCauses` as well, or is it a given that `errorCauses` would be an array?

Reply:
I think so. As per V1 - https://github.com/okta/okta-signin-widget/blob/4.4/src/util/FactorUtil.js#L349 - it seems to be. Also, the only change between V1 and OIE is that `errorSummary` is an array in OIE. One of the reasons for this PR.